packages/components/modal/src/Modal.styles.ts

```diff
@@ -0,0 +1,113 @@
+import tokens from '@contentful/f36-tokens';
+import { css, cx } from 'emotion';
+import { ModalPositionType, ModalSizeType } from './types';
+
+export function getModalStyles(props: {
+  size: ModalSizeType;
+  position: ModalPositionType;
+  allowHeightOverflow?: boolean;
+  className?: string;
+}) {
+  const modal = cx(
+    css({
+      transition: `opacity ${tokens.transitionDurationLong} ${tokens.transitionEasingDefault}, transform ${tokens.transitionDurationDefault} ${tokens.transitionEasingCubicBezier}`,
+      opacity: '0.5',
+      margin: '50px',
+      transform: 'scale(0.85)',
+      backgroundColor: tokens.colorWhite,
+      borderRadius: tokens.borderRadiusMedium,
+      boxShadow: tokens.boxShadowHeavy,
+      maxHeight: `calc(100vh - 1rem * (100 / ${tokens.fontBaseDefault}))`,
+      maxWidth: `calc(100vw - 1rem * (100 / ${tokens.fontBaseDefault}))`,
+      overflow: 'hidden',
+      display: 'flex',
+      flexDirection: 'column',
+    }),
+    props.allowHeightOverflow
+      ? css({
+          overflow: 'auto',
+          maxHeight: 'none',
+        })
+      : null,
```

Comment:
This property is exposed as a part of UI Extensions SDK, so it can be removed without breaking people apps and extensions.

https://github.com/contentful/ui-extensions-sdk/blob/b0bbaad477114f7929d974b8bce8c3c03f05f4bf/lib/types.ts#L477